bin/render-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I like it 👍 

This will make it easier to deploy becuase we can just call this command to build the app on Render